compiler/rustc_hir_analysis/src/hir_ty_lowering/errors.rs

```diff
@@ -901,6 +909,10 @@ impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
                     // `Trait<'a, Item = Type>` while accounting for the `<'a>` in the
                     // suggestion.
                     format!("{}, {}>", &snippet[..snippet.len() - 1], types.join(", "))
```

Comment:
(preexisting) just noticed that the `snippet.ends_with(`>`) is a bit flawed but it's not worth fixing (suggests `Tr<, Item = Type>` (sic!) for `<Trait<>>::func`)